config.md

```diff
@@ -56,12 +56,12 @@ This specification uses the following terms:
     </dd>
 </dl>
 
-## Image JSON Description
+## Image Configuration
```

Comment:
I [meant][1] the H1 at the top of this file.  This whole file is about that configuration, right?

[1]: https://github.com/opencontainers/image-spec/pull/305#discussion_r78646058